app/scripts/services/projects.js

```diff
@@ -38,13 +38,6 @@ angular.module('openshiftConsole')
                                           context.projectPromise.resolve(project);
                                           // TODO: fix need to return context & projectPromise
                                           return [project, context];
-                                        }, function() {
```

Comment:
You might still need an error handler here that returns the [project, context]  since we don't want it to fail resolving the project if the rules review fails.